RxJavaTemplate/buildSrc/src/main/java/DependencyHandlerExt.kt

```diff
@@ -0,0 +1,59 @@
+import org.gradle.api.artifacts.dsl.DependencyHandler
+
+private const val DEPENDENCY_API = "api"
+private const val DEPENDENCY_KAPT = "kapt"
+private const val DEPENDENCY_KAPT_ANDROID_TEST = "kaptAndroidTest"
+private const val DEPENDENCY_IMPLEMENTATION = "implementation"
+private const val DEPENDENCY_IMPLEMENTATION_DEBUG = "debugImplementation"
+private const val DEPENDENCY_IMPLEMENTATION_RELEASE = "releaseImplementation"
+private const val DEPENDENCY_IMPLEMENTATION_TEST = "testImplementation"
+private const val DEPENDENCY_IMPLEMENTATION_ANDROID_TEST = "androidTestImplementation"
+
+// Util functions for adding the different type dependencies from build.gradle file
```

Comment:
small wording improvement 
```suggestion
// Util functions for adding the different type of dependencies from build.gradle file
```